bool status
string sender
---
bool success

